lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -269,9 +289,9 @@ def main():
     new_path = module.params.get('new_path')
     dup_ok = module.params.get('dup_ok')
     if state == 'present' and not new_name and not new_path:
-        cert = module.params.get('cert')
-        key = module.params.get('key')
-        cert_chain = module.params.get('cert_chain')
+        cert, key, cert_chain = load_data(cert=module.params.get('cert'),
```

Comment:
We can check if the string is a cert easily.  Certs are always in ASCII format (AWS doesn't accept binary certificates) and PEM certs will always start with `-----BEGIN`.  That would imply we can check for this as an identifier of a Cert (vs a file path)